src/main/scala/nl.knaw.dans.easy.bagstore/server/StoresServletComponent.scala

```diff
@@ -175,8 +177,7 @@ trait StoresServletComponent extends DebugEnhancedLogging {
                 InternalServerError(s"[${ new DateTime() }] Unexpected type of failure. Please consult the logs")
             }
         })
-        .getOrElse(NotFound(s"No such bag-store: $bagStore"))
-        .logResponse
+        .getOrElse(NotFound(s"No such bag-store: $bagStore")) logResponse
```

Comment:
I still prefer the `.logResponse` notation. Note that it also minimizes the diff in this PR 😉